src/app/components/page.component.ts

```diff
@@ -73,7 +87,7 @@ export abstract class PageComponent implements OnInit, OnDestroy {
   }
 
   static isLibrary(entry) {
-    return Boolean(entry.role && entry.visibility);
+    return isLibrary(entry) || entry.nodeType === 'st:site';
```

Comment:
can this be one function instead of two? what's the purpose having extra `function isLibrary(arg: any): arg is Site` ?

Reply:
https://github.com/Alfresco/alfresco-content-app/pull/826/files#diff-64dda6fdf9138e94cf535c53e560301eR52 its a custom type guard

Reply:
I mean function 'isLibrary' is calling function 'isLibrary' in the same file.

Reply:
one is a static component method, the other is the type guard. I guess I could have changed the name to clarify that